src/linux-release-info.ts

```diff
@@ -153,68 +155,64 @@ function getOsInfo(): OsInfo {
 async function readAsyncOsReleaseFile(
   fileList: string[],
   options: LinuxReleaseInfoOptions,
-): Promise<OsInfo> {
+): Promise<Result<OsInfo>> {
   let fileData = null;
 
   for (const osReleaseFile of fileList) {
     try {
-      if (options.debug) {
-        /* eslint-disable no-console */
-        console.log(`Trying to read '${osReleaseFile}'...`);
-      }
+      if (options.debug)
+        actionsCore.debug(`Trying to read '${osReleaseFile}'...`);
```

Comment:
```suggestion
      if (options.debug)
        actionsCore.debug(`Trying to read '${osReleaseFile}'...`);
```

-1 on the braceless if's